Forjj segfault if contrib repo location does not exist

See log below.
```
[uggla@ugglalaptop bitbucket]$ rm -Rf bitbucket/ && forjj create --infra-path=bitbucket --run-plugin-debugger=bitbucket --contribs-repo=~/go/src 2>&1 | tee kevin.log
branch master, build_date 2018-07-21_22:58:28_+0200, build_commit ebbca899263e4918ed70353f51cefc902e69f10b, build_tag false.
WARNING ! main.(*Forj).ParseContext: stat /home/uggla/workspace/go/src/bitbucket/bitbucket: no such file or directory
2018/08/03 12:49:16 git init /home/uggla/workspace/go/src/bitbucket/bitbucket/.forj-workspace/deployments/production
2018/08/03 12:49:16 stdout Initialized empty Git repository in /home/uggla/workspace/go/src/bitbucket/bitbucket/.forj-workspace/deployments/production/.git/
2018/08/03 12:49:16 git init /home/uggla/workspace/go/src/bitbucket/bitbucket/.forj-workspace/deployments/dev
2018/08/03 12:49:16 stdout Initialized empty Git repository in /home/uggla/workspace/go/src/bitbucket/bitbucket/.forj-workspace/deployments/dev/.git/
2018/08/03 12:49:16 git init /home/uggla/workspace/go/src/bitbucket/bitbucket/.forj-workspace/deployments/test
2018/08/03 12:49:16 stdout Initialized empty Git repository in /home/uggla/workspace/go/src/bitbucket/bitbucket/.forj-workspace/deployments/test/.git/
2018/08/03 12:49:16 Organization : 'demoforjjprod'
2018/08/03 12:49:16 Unable to load plugin information for instance 'bitbucket'. Document not found from URLs given
WARNING ! main.(*Forj).contextDisplayed: Forjj has loaded the following context:
Forjfile model: /home/uggla/workspace/go/src/bitbucket/Forjfile
-------------------------------------

2018/08/03 12:49:16 Set default upstream application to 'bitbucket'
2018/08/03 12:49:16 CREATE: Automatic git push and forjj maintain enabled.
WARNING ! forjj/forjfile.(*Forge).Validate: Found infra data from deployment ''. This definition is ignored. Move it to your main Forjfile.
WARNING ! forjj/forjfile.(*Forge).Validate: Found infra data from deployment ''. This definition is ignored. Move it to your main Forjfile.
Validated successfully.
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x7cbf86]

goroutine 1 [running]:
forjj/scandrivers.(*ScanDrivers).DoScanDriversObject(0xc4201a3080, 0xc4201c8ec0, 0xc4201a3080)
	/go/src/forjj/scandrivers/scan_drivers.go:99 +0x666
main.(*Forj).scanAndSetDefaults(0xb897c0, 0xc42017e150, 0x8fb86f, 0x6, 0x7721da, 0xc4200b4900)
	/go/src/forjj/drivers_options.go:482 +0x136
main.(*Forj).Create(0xb897c0, 0xc42011e5a0, 0x3)
	/go/src/forjj/create.go:122 +0x142
main.(*Forj).createAction(0xb897c0, 0x8fb80f, 0x6)
	/go/src/forjj/create.go:18 +0x40
main.(*Forj).(main.createAction)-fm(0x8fb80f, 0x6)
	/go/src/forjj/app.go:239 +0x3e
main.main()
	/go/src/forjj/forjj.go:52 +0x1ed
```

--- a/create.go
+++ b/create.go
@@ -16,7 +16,7 @@ import (
 
 func (a *Forj) createAction(string) {
 	if err := a.Create(); err != nil {
-		log.Fatalf("Forjj create issue. %s", err)
+		log.Fatalf("Forjj create issue. %s.", err)
 	}
 	log.Print("===========================================")
 	if !*a.no_maintain {
@@ -115,7 +115,7 @@ func (a *Forj) Create() error {
 	}
 
 	if err := a.ValidateForjfile(); err != nil {
-		return fmt.Errorf("Your Forjfile is having issues. %s Try to fix and retry.", err)
+		return fmt.Errorf("Your Forjfile is having issues. %s. Fix it and retry", err)
 	}
 
 	// Set plugin defaults for objects defined by plugins loaded.

--- a/validate.go
+++ b/validate.go
@@ -1,8 +1,8 @@
 package main
 
 import (
-	"log"
 	"fmt"
+	"log"
 
 	"github.com/forj-oss/goforjj"
 )
@@ -26,6 +26,12 @@ func (a *Forj) FoundValidAppFlag(key, driver, object string, required bool) (_ b
 		err = fmt.Errorf("Internal issue. Driver %s not found in memory", driver)
 		return
 	}
+
+	if d.Plugin == nil {
+		err = fmt.Errorf("No %s driver '%s' loaded for application '%s'", d.DriverType, d.Name, driver)
+		return
+	}
+
 	if o, found := d.Plugin.Yaml.Objects[object]; found {
 		return o.HasValidKey(key), nil
 	}
@@ -39,7 +45,7 @@ func (a *Forj) FoundValidAppFlag(key, driver, object string, required bool) (_ b
 func (a *Forj) ValidateForjfile() (_ error) {
 	f := a.f.DeployForjfile()
 
-	if err := a.f.Validate() ; err != nil {
+	if err := a.f.Validate(); err != nil {
 		return fmt.Errorf("Validation error. %s", err)
 	}
 

